lib/keypair.ex

```diff
@@ -65,6 +66,17 @@ defmodule Elixium.KeyPair do
     :crypto.verify(@sigtype, @hashtype, data, signature, [public_key, @curve])
   end
 
+  @doc """
+    Using a public address, fetch the correct keyfile and return the only the private key
+  """
+  @spec get_priv_from_file(String.t()) :: {binary, binary}
+  def get_priv_from_file(pub) do
+    unix_address = Application.get_env(:elixium_core, :unix_key_address)
+    key_path = unix_address <> "/" <> pub <>".key"
```

Comment:
This would look cleaner if we changed it to:

```elixir
"#{unix_address}/#{pub}.key"
```

Reply:
Fixed